distributed/__init__.py

```diff
@@ -45,8 +45,8 @@
 from distributed.diagnostics.plugin import (
     CondaInstall,
     Environ,
+    InstallPlugin,
     NannyPlugin,
-    PackageInstall,
```

Comment:
We may want to consider a deprecation cycle here, but `PackageInstall` was never _truly_ public since `_PackageInstaller` was never public.

Reply:
I think that's fine